### ISSUE TITLE
From '*' doesn't work

### ISSUE THREAD

**Issue by kybarg** [vstirbu/fsm-as-promised#118]
Error is thrown, so ned to list all possible states

**Comment by vstirbu** [comment]
Yes, the wildcard `*` has been inherited from [javascript-state-machine](https://github.com/jakesgordon/javascript-state-machine) dsl and has been neglected since.

Could you give an example of the nature of the transition where you need the `from` wildcard that cannot be handled by other handlers such as `onleave` or `onenter`? What is the number of states that have to be entered manually?

Personally, I think that the `from` wildcard is an anti-pattern. I think that the definition of state machine should be implicit, but I'm interested to know more about your use case.

Cheers!
